index.html

```diff
@@ -109,7 +109,7 @@ <h2 class="module-title font-alt">About Me</h2>
                         <strong>Email Address:</strong> <a href="mailto:#">carishchinoy@gmail.com</a>
                     </address>
 
-                    <a class="btn btn-default" href="assets/Resume,%20Carishma%20Chinoy.pdf" download>
+                    <a class="btn btn-default" href="assets/Resume, Carishma Chinoy.pdf" download>

```

Comment:
you want to leave `%20` in there, it's the `url` representation of a ` ` character